src/components/FormBuilder.tsx

```diff
@@ -280,7 +287,7 @@ export class FormBuilder extends React.Component<IFormBuilderProps, {}> {
 
         const component = React.createElement(fieldDef.builder, fieldBuilderProps);
         return (
-            <div className='form-builder-field' key={field.id}>
+            <div ref={this.attachFieldRef} className='form-builder-field' key={field.id}>
```

Comment:
This change changes the UX for loading a template as well. When a template is loaded the last field will be scrolled into view automatically.